src/views/blocks/tickets/content-inactive.php

```diff
@@ -15,8 +15,7 @@
  *
```

Comment:
This view also needs @version TBD and @since TBD Use tribe_get_ticket_label_plural() for "tickets" string.